README.md

```
@@ -1,9 +1,204 @@
+
+#### 💳 결제 · 리뷰 · 배송지
+- 완료된 주문에 한해 리뷰 작성 및 가게별 평균 평점 제공
+- 배송지 관리
+
+---
+
+## 📐 ERD
+
+<div align="center">
+
+<img src="./images/erd.png" width="600"/>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)